leobot_description/robot/leobot.gazebo.xacro

```diff
@@ -127,4 +127,38 @@
     <material>Gazebo/Green</material>
   </gazebo>
 
+  <!-- Hokuyo LIDAR for simulation -->
+   <gazebo reference="laser_link">
+      <material>Gazebo/Green</material>
+      <sensor type="ray" name="laser">
+        <pose>0 0 0 0 0 0</pose>
+        <visualize>true</visualize>
```

Comment:
Please hide LIDAR visualization from Simulation. It is better seen in the RViz no need to overload simulator :)